src/utils.pl

```diff
@@ -1,31 +1,35 @@
-encoding(utf8).
+:-consult('pieces.pl').
+concatenate_lists([], List, List).
+concatenate_lists([X|Xs], List2, [X|Result]) :-
+    concatenate_lists(Xs, List2, Result).
 
-print_matrix([]).
-print_matrix([Row|Rest]) :-
-    print_row(Row),
-    nl, % Newline to separate rows
-    print_matrix(Rest).
 
 print_string_([]):-!.
 print_string_([Head | Rest]):-
         char_code(Char,Head),
         write(Char),
         print_string_(Rest).
```

Comment:
tenho ideia que se deres write a uma string ele não dá erro. O unico cuidado que precisamos de ter é de pôr com single( ') e não double( " )

Reply:
write(string)  writes a list of char codes, we work with string in the draw so we need it.

Reply:
Yes, put what if we use string with single quotes and not  double? maybe something for the future